### ISSUE TITLE
The `mix_format_file` command is not showing error details on the panel

### ISSUE THREAD

**Issue by danilokleber** [elixir-editors/elixir-sublime-syntax#65]
If formatting fails it does not show the output details for a quick reference on what to fix.

I noticed that the build `Elixir - mix format FILE` shows it just right.

Thanks for the attention.

**Comment by azizk** [comment]
Could it be that your key binding unintentionally runs `lsp-elixir`'s formatter?
Please check the console (``Ctrl+` ``) and see if it says `ElixirSyntax: Formatting 'some_file.ex'!` when you press your shortcut keys. Let's see what messages you get in the console. :slightly_smiling_face:

**Comment by azizk** [comment]
That's strange. Don't know what the problem could be atm.

**Comment by danilokleber** [comment]
Indeed. To add a bit more, I had previously uninstalled the [elixir-tmbundle](https://github.com/elixir-editors/elixir-tmbundle) package as this one serves me well enough. Then I missed some snippets from that package and reinstalled it. To my surprise the errors are showing back again. But I guess the formatter command from your package is not being used anymore. If I set the format on save setting on, I see the panel without the output like when I had just your package on. The console message also do not show the `ElixirSyntax: Formatting 'some_file.ex'!` message when using the key binding.
